records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code
+
+- **Be kind**
+  - Assume best intent
+  - Bring humanity to work - full spectrum of emotion and acknowledgement that we are humans who do have personal lives
+  - Reach out and be vulnerable
+
+- **Commit**
+  - Do what we say we will do (and communicate when things change)
+  - Keep practical details front of mind
+  - Prioritise team outcomes
+  - Be meaningfully wrong
+
+- **Collaborate**
+  - Teach, share information and decisions, learning early and often and recording this where people can find the information again
+  - Broadcast by default
+
+- **Engage**
+  - Engage in synchronous sessions
+  - Thumbs up/down in response to questions in a group
+  - Aim to ask questions first (curiosity, clean/clean-ish questions)
+  - Never hesitate to express ourselves
+
+## Things we never do
+
+- **During delivery**
+  - Land code without review
+  - Hide information or code
+  - Criticise the person instead of the code
+  - Go it alone when doing something outside of your comfort zone
+
+- **During communication**
+  - Make assumptions about others' behaviour (other than best intent)
+  - Bottle up frustrations without discussing them

```

Comment:
I got feedback that I'm overwhelming and "not assuming the best intent" although that I tried to discussed and ask questions to understand the problem. Asking questions and nice communications is when there is Trust. With some people there is no trust so I prefer to bottle up things.